crates/cli/cli/src/commands/query/mod.rs

```diff
@@ -1,6 +1,3 @@
-mod connection;
```

Comment:
The corresponding files at [`commands/query/connection`](https://github.com/informalsystems/hermes-sdk/tree/668e55080bef8e13ffffbeceba49d5be74de0ff8/crates/cli/cli/src/commands/query/connection) should be removed.